merlin-server/src/test/java/gov/nasa/jpl/aerie/merlin/server/services/ConstraintsDSLCompilationServiceTests.java

```diff
@@ -277,6 +278,31 @@ export default() => {
     );
   }
 
+  @Test
+  void testClampBy() {
```

Comment:
:large_orange_diamond: Should this be `testShiftBy` ?

Reply:
I've renamed them both (this one and the one just below) to `testShift1` and `testShift2`.